### PR TITLE
Updated profiles to fit BBMRI guidelines

### PR DESCRIPTION
**Pull request by noemide**
- Added ValueSet, CodeSystem & Extension for 
1. Data categories
2. collection types 
3. quality standards.

- Added Code "research" for use as contact type
- Updated Biobank to use ADMIN and RESEARCH in the respective contact slices & included quality extension
- Collection: Added extensions, added identifier for BBMRI ID, changed contact slincing to be like Biobank
- Updated documentation

### CODE REVIEW

File: extension/DataCategory.StructureDefinition.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "https://fhir.bbmri.de/StructureDefinition/DataCategory",
+  "name": "DataCategory",
```
Comment (alexanderkiel) [inline extension/DataCategory.StructureDefinition.json:4]: Can you provide a description here? I don't known what a data category is.

File: terminology/valueset/ValueSetBbmriCollectionTypes.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "resourceType": "ValueSet",
+  "id": "ValueSetBbmriDataCategories",
+  "url": "https://fhir.bbmri.de/ValueSet/ValueSetBbmriCollectionTypes",
+  "name": "ValueSetBbmriCollectionTypes",
```
Comment (alexanderkiel) [inline terminology/valueset/ValueSetBbmriCollectionTypes.json]: Isn't it redundant to repeat `ValueSet` where?

File: extension/collectionTypes.StructureDefinition.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "https://fhir.bbmri.de/StructureDefinition/collectionType",
```
Comment (alexanderkiel) [inline extension/collectionTypes.StructureDefinition.json]: title case

File: extension/quality-standards.StructureDefinition.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "https://fhir.bbmri.de/StructureDefinition/quality-standards",
+  "name": "quality-standards",
```
Comment (alexanderkiel) [inline extension/quality-standards.StructureDefinition.json]: naming convention

File: extension/quality-standards.StructureDefinition.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  "status": "draft",
```
Comment (alexanderkiel) [inline extension/quality-standards.StructureDefinition.json]: description

File: profile/GbaCollection.StructureDefinition.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "resourceType": "StructureDefinition",
-  "url": "http://fhir.germanbiobanknode.de/fhir/StructureDefinition/GbaCollection",
-  "name": "GbaCollection",
+  "url": "http://fhir.bbmri.de/StructureDefinition/Collection",
+  "name": "Collection",
```
Comment (alexanderkiel) [inline profile/GbaCollection.StructureDefinition.json]: description

File: terminology/codesystem/CodeSystemBbmriCollectionTypes.json [hunk 7]
```diff
@@ -0,0 +1,72 @@
+{
+  "resourceType": "CodeSystem",
+  "id": "CodeSystemBbmriCollectionTypes",
+  "url": "https://fhir.bbmri.de/CodeSystem/CodeSystemBbmriCollectionTypes",
```
Comment (alexanderkiel) [inline terminology/codesystem/CodeSystemBbmriCollectionTypes.json]: redundant `CodeSystem`

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

File: terminology/valueset/ValueSetBbmriCollectionTypes.json (hunk 2 above)
Comment (noemide) [reply terminology/valueset/ValueSetBbmriCollectionTypes.json]: Terminology resources are now named according to the not-outdated part of the de-basis IG (https://simplifier.net/guide/basisprofil-de-r4/NomenklaturderRessourcen)  "ValueSets, die alle Konzepte eines CodeSystems vollständig enthalten, tragen den selben Namen wie das CodeSystem"

File: profile/BbmriCauseOfDeath.StructureDefinition.json [hunk 8]
```diff
@@ -46,7 +46,7 @@
             "code": "Reference",
             "targetProfile": [
               "http://hl7.org/fhir/StructureDefinition/Patient",
```
Comment (alexanderkiel) [inline profile/BbmriCauseOfDeath.StructureDefinition.json]: should we use our own patient here?

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

**Review by alexanderkiel — APPROVED** [review]

**Comment by alexanderkiel** [comment]
We forgot `Condition.onsetDateTime`. So please add it.

**Comment by alexanderkiel** [comment]
Do you think we need `Condition.abatement[x]`? I would keep it simple and use only one date for now.

**Comment by noemide** [comment]
> 
> 
> Do you think we need `Condition.abatement[x]`? I would keep it simple and use only one date for now.

In my opinion, if we support onset, we should also at least have the option to use abatement. (Both are not mandatory) We also have recordedDate, for cases in which only a system timestamp is available. (Also not mandatory for cases in which no date information is present, but i will recommend using at least recordedDate or onsetDate in the textual IG)

**Comment by alexanderkiel** [comment]
Ok recordedDate is useful if one has no onsetDateTime information available. In queries we can see us using onsetDateTime with a fallback to recordedDate. But I don't see us using abatementDateTime. If we ask "get all specimen with a collectedDateTime up to x amount of days after a condition happened (onset) or was known (recorded), what should abatementDateTime help us here? Do you have an example?

**Comment by noemide** [comment]
E.g. Sample from a patient with Condition A while they also had Condition B. Concrete example: Patient who had chronic pneumonia and developed lung cancer (in contrast to having and recovering from pneumonia some time before the cancer). Im not sure if the UI supports such queries yet, but it may be useful for future applications - and since it is optional, biobanks can choose not to provide the data until it becomes relevant.

**Comment by alexanderkiel** [comment]
Ok we make all three dates optional and explain in the IG that we plan to use onset with recorded as fallback in first date-based queries and abatement maybe later. ok?

**Comment by noemide** [comment]
Sounds good :)

**Comment by alexanderkiel** [comment]
Do you plan to rename the profiles in this feature branch or in the next one?

**Comment by noemide** [comment]
I currently plan to use the next one where i already started setting mustSupport flags. (And also because we're still waiting on feedback and this may take a while.) But we can also leave this PR open until it is decided?

**Comment by alexanderkiel** [comment]
I think both issues, removing Bbmri and using Patient/Specimen is decided. I would put that changes in this PR. Maybe also the ICD-10 validation if possible. Else I would merge this PR and move on.

**Pull request merged**